Extensions/timestamps.js

```diff
@@ -170,9 +205,13 @@ XKit.extensions.timestamps = new Object({
 	},
 
 	add_reblog_timestamps: function() {
+		var preferences = XKit.extensions.timestamps.preferences;
 		var selector = ".reblog-list-item";
-		if (XKit.extensions.timestamps.preferences.only_original.value) {
+
+		if (!preferences.reblogs.value) {
```

Comment:
this makes my brain hurt.... maybe something like this is better?

```js
const selector = [
  preferences.op.value && '.reblog-list-item.original-reblog-content',
  preferences.reblogs.value && '.reblog-list-item:not(.original-reblog-content)'
].filter(x => x).join(',');
```

Reply:
also it's kind of weird that you can get into a state where you're showing *only* the later reblog timestamps and not the OP timestamp lol. maybe the OP checkbox should be grayed out and forced to true when "reblog comments" is enabled? Or it should be a three-option dropdown?